src/lib/converter/converter.ts

```diff
@@ -338,6 +338,19 @@ export class Converter extends ChildableComponent<Application, ConverterComponen
             result.notSupportedIn = tag[1].split(/,\s?/);
         }
 
+        if (result && comment != null && comment.indexOf('@examples') != -1) {
+            var examplesTagRegex = /@(?:examples)\s*((?:'[^'\\]*(?:\\.[^'\\]*)*',\s*)*'[^'\\]*(?:\\.[^'\\]*)*')/;
```

Comment:
I find this regex hard to read and understand. Could we simplify if we did not require quotes around the values and simply split on the comma?

And then typedoc could add something standard, whether it's single quotes, or double quotes which is the prettier standard we have been using inside our projects.

This will also prevent inconsistencies for future collaborators who may not use the right quotes and get errors.

Reply:
I used this (admittedly complex) regex so we could include commas (or whatever character we use as a separator) in the examples themselves.

I can't really think of a concrete case where that would be truly useful, though... so perhaps using something as simple as @(?:examples)\s*(.+) and splitting on commas would do the job.

Or maybe I could use a lookbehind regex when splitting on commas to allow commas to be escaped with `\`. I think I'll try this.